.github/workflows/yarn_test.yaml

```diff
@@ -1,49 +1,77 @@
 name: Node CI
 
-on: [push]
+on: [pull_request]
```

Comment:
sure we just want to run this on PRs? Does it take a long time to run, is that why you want to reduce it?

Reply:
Yeah, it takes a couple of minutes usually, which might be okay but I'm happy for this to run just in CI and let developers get on with their day – myself included